grafana/dashboard.json

```diff
@@ -1002,38 +933,31 @@
         "regex": "",
         "skipUrlSync": false,
         "sort": 0,
-        "tagValuesQuery": "",
-        "tagsQuery": "",
-        "type": "query",
-        "useTags": false
+        "type": "query"
       },
       {
-        "allValue": ".*",
-        "current": {},
-        "datasource": {
-          "type": "prometheus",
-          "uid": "${DS_PROMETHEUS}"
+        "current": {
+          "selected": true,
+          "text": "tv-sonarr",
+          "value": "tv-sonarr"
         },
-        "definition": "label_values(qbittorrent_torrents_count{server=\"$server\"},category)",
+        "definition": "label_values(qbittorrent_torrents_count,category)",
         "hide": 0,
-        "includeAll": true,
+        "includeAll": false,
         "label": "Categories",
-        "multi": true,
+        "multi": false,
```

Comment:
This is also important or you will see only a specific category in the dashboard